sw/host/opentitanlib/src/transport/common/fpga.rs

```diff
@@ -45,9 +46,26 @@ impl FpgaProgram<'_> {
         Ok(false)
     }
 
-    pub fn skip(&self) -> bool {
+    fn skip(&self) -> bool {
         self.bitstream.starts_with(b"__skip__")
     }
+
+    pub fn should_skip(&self, transport: &TransportWrapper) -> Result<bool> {
+        // Open the console UART.  We do this first so we get the receiver
+        // started and the uart buffering data for us.
+        let uart = transport.uart("CONSOLE")?;
+        let reset_pin = transport.gpio_pin("RESET")?;
+        if self.skip() {
+            log::info!("Skip loading the __skip__ bitstream.");
+            return Ok(true);
+        }
+        log::warn!("Checking version");
```

Comment:
nitpick: I don't think this should be a warning, more like an info or debug.

Reply:
This should not even be debug, this section of code is moved from the `dispatch()` method of `ChipWhisperer`.   At some point while developing, I must have added inserted extra logging for my own debugging, and forgotten them there.  I do not think they serve any purpose, and have removed this, and the one referred below.